package/requirements.txt

```diff
@@ -15,6 +15,7 @@ numpy>=1.20.0
 packaging
 parmed
 pytest
+pyyaml>=3.02
```

Comment:
Why the pyyaml addition?

Reply:
I added this constraint to all files that check for the sphinx installation since this constraint allows the citations to be compiled.

Reply:
Im surprised the docs arn't working normally... 

Perhaps fix the CHANGELOG conflicts and re-trigger CI and I'll have a look.

Reply:
So to be clear, you want me to remove this change and then run the CI so you can see the error? I'm 90% sure that the failure in https://github.com/MDAnalysis/mdanalysis/pull/3905 is this same failure, so I could also add this change to that PR to show that it's resolved.

Reply:
@hmacdope I tried to look through my browsing history to find the cite where I found the answer, but I didn't find it. I thiiiink I took a shot in the dark based on this statement (https://avocado-framework.readthedocs.io/en/latest/releases/44_0.html?highlight=pyyaml#documentation) and it worked.

Here is the link to the check-logs for this PR where you see the same errors that are now presenting in #3905, illustrating that the issue is consistent between the two.<https://github.com/MDAnalysis/mdanalysis/actions/runs/3159210367/jobs/5142154816>

Also I did give a heads up about this to the citation work that is under development here:
https://github.com/MDAnalysis/UserGuide/issues/212

Reply:
Sorry I'm coming here a bit blind to this whole issue, but why `requirements.txt`? Is this for an issue with running things locally?

Re: CI, as far as I'm aware either uses optional deps defined in the github action or the environment.yaml file under maintainer.

Reply:
It was for an issue with compiling the docs with CI, and I can't remember if I had the same issue locally. Although it appears that this is not longer an issue since the other PR with the same issue compiled the docs in CI (#3905).

I'll remove the commits that added these dependencies.

Reply:
Ok in theory I should be able to remove the two commits involved in this thread (first to add the dependencies and then to remove them). However, I keep making a mess, could I get some advice?

1) `git rebase -i HEAD~30`, this shows a list of all previous commits from both me and the develop branch.
2) At the commits of interest, I change "pick" to "drop" and then save and quit in vi
3) A conflict in the CHANGELOG appears (like always) and asks me to change it. If I do, git adds the change to someone else's commit, so instead I type `git rebase --skip` a million times and the conflict is eventually resolved down the line.
4) I can then force push these changes to GitHub, however, all commits from the development branch now show up on this PR with the label "@SoAndSo authored, @jaclark5 committed". I don't like this so then I reflog locally to undo everything and force push that. 

Is there something I should be doing differently?

Reply:
We usually squash merge PRs, so history doesn't usually end up being an issue, would this fix things here?

Reply:
Hey, if you mean that squashing the commits will cancel out the two that I want to remove, then yes :). Ok I think this is good to go then if I've addressed all of @hmacdope's concerns (I think I have).